src/docs/PML/PMLUserGuide.adoc

```diff
@@ -1564,6 +1564,130 @@ This example trains a Mnist model, starts a service instance where the trained m
 
 image::NEW_MAAS_DL_MNIST_Workflow_Example.PNG[align=center]
 
+== AutoFeat
+
+The performance of a machine learning model not only depends on the model and the hyper-parameters but also on how we process and feed different types of variables to the model.
+
+Before going for the modelling, there are various tasks we require to perform as a part of the data preparation. Encoding categorical data is one of such tasks which is considered crucial. As we know, most of the data in real life come with categorical string values and most of the machine learning models basically perform mathematical operations. But the harsh truth is that mathematics is totally dependent on numbers. So in short we can say that most of the machine learning models only accept numerical variables, not strings and these numbers can be float or integer. Thereafter, preprocessing and encoding the categorical variables becomes a necessary step, such we need to convert these categorical variables to numbers which can help in predicting the outcomes in a machine learning task.
+
+AutoFeat provides a complete solution to assist data scientists to encode successfully their categorical data.
+
+In real-world problems, most of the time we require choosing one encoding method for the proper working of the model. Working with different encoders can vary the results of the model.
+
+AutoFeat currently supports the following encoding methods:
```

Comment:
do you have a reference ? for these techniques ?

Reply:
Yes, i added the reference for the python library (category encoders) that proposed an implementation of those techniques. Do you suggest to add also the reference for each of those techniques?